fetch.bs

```diff
@@ -564,6 +572,22 @@ A: 3
  <var>name</var> and <a for=header>value</a> is <var>value</var> to <var>list</var>.
 </ol>
 
+<p>To <dfn export for="header list" id=concept-header-list-set-structured-header>set a structured
+header</dfn> <a for=header>name</a>/<a>structured header</a>
```

Comment:
I filed https://github.com/httpwg/http-extensions/issues/908 as this reads rather awkward.